integration/container/attach_test.go

```diff
@@ -0,0 +1,206 @@
+// +build !windows
+
+package container
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"io"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/integration/internal/container"
+	"gotest.tools/assert"
+	is "gotest.tools/assert/cmp"
+	"gotest.tools/poll"
+	"gotest.tools/skip"
+)
+
+// #9860 Make sure attach ends when container ends (with no errors)
+func TestAttachClosedOnContainerStop(t *testing.T) {
+	skip.If(t, testEnv.IsRemoteDaemon, "test requires daemon on the same host")
```

Comment:
Wondering why this test requires a local daemon 🤔 I see this was added in https://github.com/moby/moby/commit/a943c401509e7994ae5c574a4b7e23354e44a105 (https://github.com/moby/moby/pull/20218) but I don't see a clear explanation for _this_ particular test to require a local daemon.

Reply:
Thank you for the review. I have removed the skip, as I could not find a reason for requiring a local daemon for this.